### PR TITLE
Implement a new feature that allows extending a running maintenance

### LINKED ISSUE #8: Implement a new feature that allows extending a running maintenance
Original issue opened by @ArturoQuintana at https://github.com/kytos/maintenance/issues/39.

@ajoaoff @rmotitsuki @italovalcy @hdiogenes 
During maintenance, new problems can arise that might not be contemplated as part of the process. These cases use to increase the maintenance windows (mostly). Therefore, it will be useful to have the capability of extending running maintenances.

### PR DESCRIPTION
**Pull request by ajoaoff**
This PR fixes #8.

Description of the change
-----------------------------
A new endpoint was created to extend a running maintenance. The API documentation and the tests were also updated.

Release notes
---------------
- New REST API endpoint to extend a running maintenance.

### CODE REVIEW

File: openapi.yml [hunk 1]
```diff
@@ -155,8 +155,37 @@ paths:
...
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                minutes:
+                  type: integer        
+      responses:
+        '201':
```
Comment (ArturoQuintana) [inline openapi.yml:181]: '200':

**Review by ArturoQuintana — CHANGES_REQUESTED** [review]
Keep a coherent relationship between the response status code.

File: openapi.yml (hunk 1 above)
Comment (ajoaoff) [reply openapi.yml]: Solved in the new commit

**Review by italovalcy — COMMENTED** [review]
Hi @ajoaoff I don't know if I missed something, but this PR is just for the documentation, right? From the description, I was expecting to have the actual code that implements the new feature of extending the running MW

**Comment by ajoaoff** [comment]
> Hi @ajoaoff I don't know if I missed something, but this PR is just for the documentation, right? From the description, I was expecting to have the actual code that implements the new feature of extending the running MW

I made some mistake when I last updated the `openapi.yml` file and the changes in `main.py` just vanished.

**Comment by ArturoQuintana** [comment]
Hi @ajoaoff, that's maybe the reason the test is not working properly.

**Review by ArturoQuintana — APPROVED** [review]

**Review by ArturoQuintana — APPROVED** [review]

**Review by ArturoQuintana — APPROVED** [review]

**Pull request merged**